kibbeh/src/stories/UserWideButton.story.tsx

```diff
@@ -0,0 +1,25 @@
+import { Story } from "@storybook/react";
+import React from "react";
+import { sampleBaseUser } from "./data/BaseUser";
+import { UserWideButton, UserWideButtonInfoProps } from "../ui/UserWideButton";
+
+export default {
+  title: "UserWideButton",
+  component: UserWideButton,
+};
+
+const changeText = (isBlocked: boolean, isFollowing: boolean, text: string) => {
+    if(isBlocked){
+        return "Unblock";
+    } else if (isFollowing){
+        return "Follows you"
+    } else{
+        return "block"
```

Comment:
Semicolon missed
```
return "block";
```

Reply:
And `"block"` has lowercase first letter, but another phrases starts with uppercase